File: app/src/externalRelease/java/com/kickstarter/libs/NoopBuildCheck.java
package com.kickstarter.libs;

import com.kickstarter.presenters.DiscoveryPresenter;
import com.kickstarter.services.WebClient;

public class NoopBuildCheck implements BuildCheck {
  @Override
  public void bind(final DiscoveryPresenter presenter, final WebClient client) {
    // No-op, distribution through Play Store
  }
}

File: app/src/androidTest/java/com/kickstarter/screenshoot/testing/di/NoopBuildCheck.kt
package com.kickstarter.screenshoot.testing.di

import androidx.annotation.NonNull
import com.kickstarter.libs.BuildCheck
import com.kickstarter.services.WebClientType
import com.kickstarter.viewmodels.DiscoveryViewModel

class NoopBuildCheck : BuildCheck {
    @Override
    override fun bind(@NonNull viewModel: DiscoveryViewModel.ViewModel, @NonNull client: WebClientType) {
        // No-op
    }
}
